frontend/src/features/properties/components/forms/subforms/AddressForm.tsx

```diff
@@ -73,21 +74,23 @@ const AddressForm = <T extends any>(props: AddressProps & FormikProps<T>) => {
 
   return (
     <>
-      <Row className="field-row">
-        <Label required={!props.verticalLine}>Street Address</Label>
-        {props.verticalLine && <span className="vl"></span>}
-        <GeocoderAutoComplete
-          tooltip={props.toolTips ? streetAddressTooltip : undefined}
-          value={getIn(props.values, withNameSpace('line1'))}
-          disabled={props.disabled}
-          field={withNameSpace('line1')}
-          onSelectionChanged={handleGeocoderChanges}
-          onTextChange={value => props.setFieldValue(withNameSpace('line1'), value)}
-          error={getIn(props.errors, withNameSpace('line1'))}
-          touch={getIn(props.touched, withNameSpace('line1'))}
-          displayErrorTooltips
-        />
-      </Row>
+      {props.hideStreetAddress !== true && (
```

Comment:
This form is essentially the same between bare land, associated land and buildings except for the display of the address line.